docs/content/component-data.md

```diff
@@ -9,56 +9,33 @@ SpatialOS components are generated from `.schema` files into components that the
 ### Overview
 
 A `struct`, which implements `Unity.Entities.IComponentData` and `Improbable.Gdk.Core.ISpatialComponentData`,
-is generated for each [blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) SpatialOS component.
-A `class`, which extends `UnityEngine.Component` and implements `Improbable.Gdk.Core.ISpatialComponentData` is generated for each non-blittable SpatialOS component.
+is generated for each SpatialOS component.
 
 Each of these structs will be named `SpatialOS[schemalang component name]` and will contain only the schema data fields.
 It will **Not** contain any fields or methods pertaining to [commands](commands.md) or [events](events.md) defined on that component. 
```

Comment:
"It will **Not**...should carry on from the sentence above, with no new para. Also, best to use italics for stress.
Suggest "relating" rather than "pertaining" as it's less formal, more accessible.
Suggest:

"They do _not_ contain any fields or methods relating to [commands](commands.md) or [events](events.md) defined on that component."